app/components/verkiezingen/draft-mandataris-list.js

```diff
@@ -62,52 +63,6 @@ export default class DraftMandatarisListComponent extends Component {
     this.mandatarissen = this.store.query('mandataris', queryParams);
```

Comment:
Just saw this because it throw an error  on my logic but there is an await missing.. Updated this on my new branch. Did you have any issues with this? cc: @bfidlers